services/frontend-service/src/ui/components/ServiceLane/Warnings.scss

```diff
@@ -0,0 +1,7 @@
+.service-lane__warning {
```

Comment:
There is no license header in this file.

Reply:
true. We generally don't have that in scss files.
I will create a separate PR to add it.